Client/Application/LeanplumIntegration.swift

```diff
@@ -122,13 +123,13 @@ class LeanplumIntegration {
     // Events
 
     func track(eventName: LeanplumEventName) {
-        if shouldSendToLP() {
+        if let isInPrivateMode = profile?.prefs.boolForKey("isInPrivateMode"), !isInPrivateMode && shouldSendToLP() {
```

Comment:
Nit: Try to reference a constant for the key instead of the string since we use it more than once. Saves us from any kind of typo errors that might happen